src/pymodaq_gui/managers/action_manager.py

```diff
@@ -204,8 +213,8 @@ def add_action(self, short_name: str = '', name: str = '', icon_name: str = '',
             the name as referenced in the dict self.actions
         name: str
             Displayed name if should be displayed in
-        icon_name: str
-            png file name to produce the icon
+        icon_name: str / QtGui.QIcon
```

Comment:
str or Path

maybe explain below that it should be either the name of the png in the icon library folder or the name of the ThemeIcon enum.